src/meta.rst

```diff
@@ -89,7 +89,7 @@ dependency and rebuild the package you should increase the build number.
 When the package version changes you should reset the build number to ``0``.
 
 
-Single Verion, Externally Managed
+Single Version, Externally Managed
 ---------------------------------
```

Comment:
Minor detail, but we should make sure to keep `-`s of the same length as the title. Addressing in PR ( https://github.com/conda-forge/conda-forge.github.io/pull/395 ).